.travis.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         - ./bin/install-test-framework.sh
       script:
         - yarn lint:js
-        - yarn test
+        - yarn test:all
 
     - name: "Floating Dependencies"
       script:
```

Comment:
Probably want to have `yarn test:all` below too :)

Reply:
Thank you for point it out :) !!